test/hook.test.js

```diff
@@ -1,10 +1,18 @@
 'use strict'
 
-const test = require('tap').test
+const { test, before } = require('tap')
 const noop = () => {}
-const plugin = require('../').internals.factory
+const fastifyBearerAuth = require('..')
 const key = '123456789012354579814'
 const keys = { keys: new Set([key]) }
+const Fastify = require('fastify')
+let plugin
+
+before(async function () {
+  const fastify = new Fastify({})
+  await fastify.register(fastifyBearerAuth, { addHook: false })
+  plugin = fastify.verifyBearerAuthFactory
```

Comment:
Better move to a separate file instead of using hack like this.

Reply:
I renamed hook.test.js to verifyBearerAuthFactory.test.js as it is not about hooks but testing the verifyBearerAuthFactory.

Does it resolve your remark?

Reply:
I means `verifyBearerAuthFactory` should placed in separate file.
So, you can do `const verifyBearerAuthFactory = require('new file')` directly.